packages/core/actions/account-client/prepareMultiUserOpRequest.ts

```diff
@@ -0,0 +1,239 @@
+import {
+    AccountOrClientNotFoundError,
+    type UserOperation,
+    estimateUserOperationGas,
+    parseAccount
+} from "permissionless"
+import type {
+    PrepareUserOperationRequestParameters,
+    PrepareUserOperationRequestReturnType,
+    SponsorUserOperationReturnType
+} from "permissionless/actions/smartAccount"
+import type { StateOverrides } from "permissionless/types/bundler"
+import type {
+    ENTRYPOINT_ADDRESS_V07_TYPE,
+    EntryPoint,
+    GetEntryPointVersion
+} from "permissionless/types/entrypoint"
+import type {
+    Abi,
+    Address,
+    Chain,
+    Client,
+    Hex,
+    LocalAccount,
+    Transport
+} from "viem"
+import { estimateFeesPerGas, getChainId } from "viem/actions"
+import type { Prettify } from "viem/chains"
+import { type EncodeDeployDataParameters, getAction } from "viem/utils"
+import type { KernelSmartAccount } from "../../accounts"
+import { getMultiUserOpDummySignature } from "../../accounts/kernel/utils/plugins/multi-chain/getMultiUserOpDummySignature.js"
+
+export type SmartAccount<
```

Comment:
why does this type need to be defined again? Can use from `permissionless` itself